### PR TITLE
Creating items

### PR DESCRIPTION
**Pull request by Noelryn**
## PT Link
https://www.pivotaltracker.com/story/show/160894691

## User Stories
 As a user
 In order to be able to choose a recipe,
 I would like to see a list of items to choose from

## Tasks
- Create unit tests
- Create feature tests
- Create item model 
- Create item controller

screenshot 2018-10-04 at 13 31 32

### CODE REVIEW

File: features/step_definitions/basic_steps.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Given("the following items exists") do |table|
+    pending
+end
+  
+  Given("I visit the landing page") do
+    visit root_path
+end
```
Comment (MiriamSal) [inline features/step_definitions/basic_steps.rb:9]: Fix indentation

**Review by MiriamSal — CHANGES_REQUESTED** [review]

File: features/step_definitions/basic_steps.rb (hunk 1 above)
Comment (MiriamSal) [reply features/step_definitions/basic_steps.rb:9]: I fixed it on my branch

Comment (Noelryn) [reply features/step_definitions/basic_steps.rb:9]: Fixed

**Review by RhonGabriel — APPROVED** [review]

**Review by MiriamSal — APPROVED** [review]

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'items#index'
```
Comment (faraznaeem) [inline config/routes.rb]: is this how we write our routes?

File: coverage/.last_run.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "result": {
+    "covered_percent": 81.82
+  }
+}
```
Comment (faraznaeem) [inline coverage/.last_run.json]: why isn't the coverage in .gitignore?

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: coverage/.last_run.json (hunk 3 above)
Comment (Noelryn) [reply coverage/.last_run.json]: Have added it in to gitignore

File: config/routes.rb (hunk 2 above)
Comment (Noelryn) [reply config/routes.rb]: So I am not sure how to fix this if I change it the tests all fail.

**Comment by faraznaeem** [comment]
Please provide a screenshot to the view file

File: app/views/items/index.html.haml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+%h1 Ingredients Matcher
+%p Select some items and we'll give you some recipes
+
+%section.container
+%h4 What's in your fridge?
+- @items.each do |item|
+    %div.item-tile{id: dom_id(item)}
+    %p= item.name
+:Checkbox
```
Comment (faraznaeem) [inline app/views/items/index.html.haml]: Are you in scope with the checkbox?

**Review by faraznaeem — CHANGES_REQUESTED** [review]

File: app/views/items/index.html.haml (hunk 4 above)
Comment (Noelryn) [reply app/views/items/index.html.haml]: Sorry this was a test which I forgot to get rid of, I am now scoped

**Comment by Noelryn** [comment]
Added in Screen shot like requested

**Comment by Noelryn** [comment]
Also scoped down the code for the check boxes

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**